cli.js

```diff
@@ -37,10 +39,10 @@ if (args._[0] !== '-') input = fs.createReadStream(args._[0]);
 
 input.pipe(utils.concat(function(input) {
   if (args.i) {
-    var newMarkdown = toc.insert(input.toString());
+    var newMarkdown = toc.insert(input.toString(), { keepDiacritics: args.keepDiacritics });
```

Comment:
here I think we should just pass `args`, so that all the options would be passed. Would you mind making that change?